src/averageAges.js

```diff
@@ -2,10 +2,12 @@
 
 /**
  * Implement calculateMenAverageAge function
- * It should return average age of men in array
- * If `century` is specified then calculate only for men living in this century
+ *
+ * Function returns average age of men in array
```

Comment:
Add line break only
when 
you reach 80
characters. Line breaks in random places 
are hard to 
read. Like in this example :)